plugins/client/io.github.flaxoos/ktor-client-circuit-breaker/3.0/install.kt

```diff
@@ -1,17 +1,8 @@
-/*
- * Copyright 2014-2024 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
- */
-
 import io.github.flaxoos.ktor.client.plugins.circuitbreaker.CircuitBreakerName.Companion.toCircuitBreakerName
```

Comment:
Please configure copyright in IntelliJ and run action to generate copyright back for the repo

Reply:
I'm not sure we want to include the copyright for the plugin sample code, do we?  I think the code within each plugin ought to be owned by the contributor and licensed based on the manifest.